src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -103,15 +106,16 @@ private <T> List<T> runSnapshotBuilderQuery(
       Query query,
       Snapshot snapshot,
       AuthenticatedUserRequest userRequest,
+      Map<String, String> userEnteredData,
```

Comment:
I would also make this `Map<String, ?> paramMap`
```suggestion
      Map<String, ?> paramMap,
```